pele_platform/enzyme_engineering/saturated_mutagenesis.py

```diff
@@ -137,26 +140,25 @@ def postprocessing(self, job):
             Output job parameters.
         """
         output_path = os.path.join(job.pele_dir, job.output)
-        reports_path = os.path.join(output_path, "*",
-                                    "{}*".format(job.report_name))
-        trajectory_path = os.path.join(output_path, "*", "trajectory*pdb")
+        reports_path = os.path.join(
+            output_path, "[!equilibration]*", "{}*".format(job.report_name)
```

Comment:
I think it is safer if we retrieve folder using the following pattern:
`os.path.join(output_path, "[0-9]*", "{}*".format(job.report_name))`